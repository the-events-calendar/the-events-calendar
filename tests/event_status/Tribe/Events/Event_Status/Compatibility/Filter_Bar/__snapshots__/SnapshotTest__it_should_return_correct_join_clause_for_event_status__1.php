<?php return 'LEFT JOIN wp_postmeta AS tribe_event_status_filterbar_alias
			ON ( wp_posts.ID = tribe_event_status_filterbar_alias.post_id
			AND tribe_event_status_filterbar_alias.meta_key = \'_tribe_events_status\' )';
