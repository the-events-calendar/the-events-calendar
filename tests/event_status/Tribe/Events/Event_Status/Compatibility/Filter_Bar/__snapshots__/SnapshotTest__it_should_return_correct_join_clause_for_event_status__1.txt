LEFT JOIN test_postmeta AS tribe_event_status_filterbar_alias
			ON ( test_posts.ID = tribe_event_status_filterbar_alias.post_id
			AND tribe_event_status_filterbar_alias.meta_key = '_tribe_events_status' )